### DISCUSSION TITLE
Multiple User type Sign-up

### DISCUSSION THREAD

**Discussion by abdu1-wahab** [supabase/supabase#17850] [Questions]
I have a web app with multiple user types i.e. teacher, student, and parent. 

Now I am using triggers to insert data from metadata to each table while signing up But the issue I am facing is I am unable to select auth table while creating a trigger. In fact, I can't see any other schema than public. 

Look forward to hearing from anyone that can help.

Thanks, Abdul

**Comment by GaryAustin1** [comment]
You have to use a 'security definer' function to get to auth schema.  The user management starter guides like in the SQL editor examples shows doing a trigger with one.
